gallery/src/components/NxStatefulTabs/NxStatefulTabsPage.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryTile } from '../../gallery-components/GalleryTiles';
+import CodeExample from '../../CodeExample';
+
+import NxTabsSimpleExample from './NxStatefulTabsSimpleExample';
+
+const tabsSimpleExampleCode = require('!!raw-loader!./NxStatefulTabsSimpleExample').default;
+
+export default function NxTablePage() {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <h3 className="nx-h3">NxStatefulTabs</h3>
+
```

Comment:
This is already present as the `.nx-page-title`.
```suggestion
```

Reply:
6fe2aeb